libraries/app/database_api.cpp

```diff
@@ -624,6 +625,54 @@ vector<optional<account_object>> database_api_impl::get_accounts(const vector<ac
    return result;
 }
 
+vector<limit_order_object> database_api::get_account_limit_orders( const string& name_or_id, const string& base, const string& quote, fc::time_point_sec expire, uint32_t limit)
+{
+   return my->get_account_limit_orders( name_or_id, base, quote, expire, limit );
+}
+
+vector<limit_order_object> database_api_impl::get_account_limit_orders( const string& name_or_id, const string& base, const string& quote, fc::time_point_sec expire, uint32_t limit)
+{
```

Comment:
Please check if `limit` is too big. Since `limit_order` is relatively small, I think we can return a bit more per page.